bin/build_container_image

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+REGISTRY=${REGISTRY:-"registry.gitlab.com"}
```

Comment:
I thought we were still going to use the docker registry (at least until they kill it).  I think we have too much pointing to docker.io right now that I don't think we should change it.

Reply:
I can't add any users to our docker.io org, we're already using 5 of 3 seats.  So, I thought it would be good to try out the Gitlab registry with this image before moving everything else over.

Reply:
I can adjust the users - that's the easy part.  That being said, the bot is already a user and should be the one we use (it's the same one we use for the build).

Reply:
Switched to docker.io with the bot user and a token